src/jose/jwk.ts

```diff
@@ -1,47 +1,8 @@
-import * as ed25519 from './algorithms/ed25519';
-import * as secp256k1 from './algorithms/secp256k1';
-import Ajv from 'ajv';
-import type { JwkEd25519Private, JwkEd25519Public } from './algorithms/ed25519';
 import type { JwkSecp256k1Private, JwkSecp256k1Public } from './algorithms/secp256k1';
+import type { JwkEd25519Private, JwkEd25519Public } from './algorithms/ed25519';
 
-/**
- * A supported private key in JWK format.
- */
-export type JwkPrivate = JwkSecp256k1Private | JwkEd25519Private;
+export { generateKeyPair as generateSecp256k1Jwk } from './algorithms/secp256k1';
+export { generateKeyPair as generateEd25519Jwk } from './algorithms/ed25519';
```

Comment:
For my understanding, you made a point to import then re-export here as opposed to letting the tests/user importing from respective algorithm files.  Is there a convention that I should be aware of or this is purely preference?

Definitely want to pick my fight, and this one is not worth it, but for context the way I saw `jwk.ts` originally is a container for generic JWK functions, as opposed to algorithm specific ones. This aligns with the intent of `jws.ts`. If one wishes to use algo capabilities, import those directly from those files.

Reply:
Purely preference. I mentioned it in my initial review of #30 here: [here](https://github.com/TBD54566975/dwn-sdk-js/pull/30#pullrequestreview-958697434). Basically just a convenience. I could go either way